client/my-sites/checkout/composite-checkout/types/transaction-endpoint.ts

```diff
@@ -153,6 +154,9 @@ export function createTransactionEndpointRequestPayloadFromLineItems( {
 	storedDetailsId: string;
 	name: string;
 } ): WPCOMTransactionEndpointRequestPayload {
+	const urlParams = new URLSearchParams( window.location.search );
```

Comment:
TIL about `URLSearchParams`! Very cool. As it's not available in MSIE, is that a problem? I see that it's used elsewhere in Calypso so maybe it's fine but thought it worth mentioning.

Reply:
Good point about MSIE. `URLSearchParams` is used in a handful of places in Calypso, so I tend it think it's probably okay.